Cargo.lock

```diff
@@ -450,26 +450,26 @@ version = "0.11.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "e68a5a99c65851e7be249f5cf510c0a136f18c9bca32139576d59bd3f577b043"
 dependencies = [
- "hmac",
+ "hmac 0.12.1",
  "pbkdf2",
  "rand",
- "sha2",
+ "sha2 0.10.8",
  "unicode-normalization",
  "zeroize",
 ]
 
 [[package]]
 name = "bip32"
-version = "0.5.1"
+version = "0.6.0-pre.1"
```

Comment:
Why the preview version? I was unable to find any documentation of what changed in the preview here: https://github.com/iqlusioninc/crates/blob/bip32/v0.6.0-pre.1/bip32/CHANGELOG.md (or in corresponding files for the RustCrypto dependencies).

Reply:
Oh I see, it's to get https://github.com/iqlusioninc/crates/pull/1254

Reply:
Depending on all of these previews and rcs does not seem ideal.
* hmac 0.13.0-pre.4
* ripemd 0.2.0-pre.4
* sha2 0.11.0-pre.4
* digest 0.11.0-pre.9
* block-buffer 0.11.0-rc.3
* crypto-common 0.2.0-rc.1

I realize it's only temporary, and that you asked @tony-iqlusion for a backport [here](https://github.com/iqlusioninc/crates/pull/1254#issuecomment-2555525936).

Reply:
Here's a `bip32` v0.5.3 backport release, however be aware it uses an older version of the `secp256k1` crate (v0.27) to retain compatibility with other v0.5.x releases: https://github.com/iqlusioninc/crates/pull/1261/files

Reply:
I'd hoped that a backport might suffice, but it turned out to be necessary to use `secp256k1 0.29` because the firmware already did, and duplicate C bindings would be a nightmare.

Reply:
Note v0.30 is out and we'll bump to that (at least) before a final release

Reply:
So maybe we should ask Keystone to bump the firmware to `secp256k1` 0.30?

@tony-iqlusion when you say "at least", is that just because there might be another release after 0.30 that you could opportunistically upgrade to, or might it be necessary? (At a glance, 0.30 was [tagged only last week](https://github.com/rust-bitcoin/rust-secp256k1/releases/tag/secp256k1-0.30.0) but there are some [significant changes since that tag](https://github.com/rust-bitcoin/rust-secp256k1/compare/379e1287445024fca71d1416c44e1a17be1c81c2...master), basically updating from `rustsecp256k1_v0_10_0_*` to `rustsecp256k1_v0_11_*`.)

Reply:
Ideally we'd bump to whatever the latest version is prior to a final release

Reply:
Our problem has apparently been resolved by https://github.com/zcash/zcash-devtool/pull/78, so all good.